pnpm-lock.yaml

```diff
@@ -1,27 +1,28 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
```

Comment:
@nbsp not very helpful, I know, but could you update your pnpm version to v9 so that this doesn't get recreated for every PR ?

Reply:
oh oops, will do